### ISSUE TITLE
2.0 beta2 :iron-a11y-keys-behavior.html:419 Uncaught TypeError: Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by sowsan** [Zecat/paper-stepper#40]
Look like 2.0 beta2 has the same issue mentioned in issue #12 

![image](https://cloud.githubusercontent.com/assets/11544153/17492544/3fa37a4c-5d7a-11e6-9b7f-499b4143db5f.png)

iron-a11y-keys-behavior.html:419 Uncaught TypeError: Cannot read property 'length' of undefined

**Comment by Zecat** [comment]
[Line 419](https://github.com/PolymerElements/iron-a11y-keys-behavior/blob/master/iron-a11y-keys-behavior.html#L419) of the last iron-a11y-keys-behavior doesn't seem to match `Cannot read property 'length' of undefined` so I guess you use an old version and maybe face [this issue](https://github.com/PolymerElements/iron-a11y-keys-behavior/pull/47). Can you show your bower dependencies?
@jamesmgg you triggered this issue before can you reproduce now?

**Comment by jamesmgg** [comment]
I'm currently not using paper-stepper anymore, so I haven't tested it... All I know is that not using shadow DOM (and using Shady DOM instead) fixes the error.

**Comment by sowsan** [comment]
FYI - issue is there with 2.0 beta3 as well.

**Comment by Zecat** [comment]
Can you show your bower.json file ?

**Comment by sowsan** [comment]
{
  "name": "paper-stepper",
  "version": "2.0-beta.3",
  "authors": [
    "Zecat jullienfelix@gmail.com"
  ],
  "description": "Material paper-stepper element.",
  "keywords": [
    "web-component",
    "polymer",
    "seed"
  ],
  "main": "paper-stepper.html",
  "license": "http://polymer.github.io/LICENSE.txt",
  "homepage": "https://github.com/zecat/paper-stepper/",
  "ignore": [
    "/._",
    "/test/"
  ],
  "dependencies": {
    "polymer": "Polymer/polymer#^1.2.0",
    "paper-button": "PolymerElements/paper-button#^1.0.11",
    "iron-icons": "PolymerElements/iron-icons#^1.1.3",
    "paper-styles": "PolymerElements/paper-styles#^1.1.4",
    "paper-ripple": "PolymerElements/paper-ripple#^1.0.5",
    "iron-selector": "PolymerElements/iron-selector#^1.3.0",
    "iron-icon": "PolymerElements/iron-icon#^1.0.8",
    "iron-flex-layout": "PolymerElements/iron-flex-layout#^1.3.1",
    "neon-animation": "PolymerElements/neon-animation#^1.1.1",
    "iron-validatable-behavior": "PolymerElements/iron-validatable-behavior#^1.0.5",
    "iron-collapse": "PolymerElements/iron-collapse#^1.2.0"
  },
  "devDependencies": {
    "paper-input": "PolymerElements/paper-input#^1.1.10",
    "paper-material": "PolymerElements/paper-material#^1.0.6",
    "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
    "web-component-tester": "_",
    "iron-form": "PolymerElements/iron-form#^1.0.16",
    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#^1.2.2"
  }
}

**Comment by sowsan** [comment]
In my bower.json , im using the below to get the paper-stepper

```
"paper-stepper": "Zecat/paper-stepper#2.0-beta.3"
```
